src/actions/etherspotActions.js

```diff
@@ -260,25 +268,61 @@ export const reserveEtherspotEnsNameAction = (username: string) => {
   };
 };
 
-export const setStableTokens = () => {
+export const fetchDefaultTokens = () => {
   return async (dispatch: Dispatch, getState: GetState) => {
     const activeAccount = activeAccountSelector(getState());
+    const {
+      session: {
+        data: { isOnline },
+      },
+    } = getState();
 
-    if (!activeAccount) {
+    if (!activeAccount || !isOnline) {
       return;
     }
 
     const supportedChain = getSupportedChains(activeAccount);
 
-    const tokens = await Promise.all(supportedChain.map((chain) => etherspotService.getStableAssets(chain)));
+    dispatch({ type: FETCHING_DEFAULT_LIST, payload: true });
+
+    // For Get Default Token List
+    const chainTokens = await Promise.all(
+      supportedChain?.map((chain) => etherspotService.getTokenListTokens(chain, ETHERSPOT_DEFAULT_LIST)),
+    );
 
-    const filteredArr: any[] = [];
+    // For Get Default Stable List
+    const chainStableTokens = await Promise.all(
+      supportedChain?.map((chain) => etherspotService.getTokenListTokens(chain, ETHERSPOT_DEFAULT_STABLE_LIST)),
+    );
 
-    await tokens.forEach((item: any) => {
-      item && filteredArr.push(...item);
+    const tokens: any[] = [];
+    await chainTokens?.forEach(async (item: any) => {
+      if (item) {
+        tokens.push(...item);
+        await getExchangeRates(item[0].chain, item, async (rates) => {
+          if (rates) await dispatch(updateRatesAction(item[0].chain, rates));
+        });
+      }
+    });
+
+    const stableTokens: any[] = [];
+    await chainStableTokens?.forEach((item: any) => {
```

Comment:
The same thing here, please use a `for` loop if you need to use async / await in a loop.